### PR TITLE
Specify OS X support in the podspec.

### PR DESCRIPTION
**Pull request by neonichu**

### CODE REVIEW

File: PeerKit.podspec [hunk 1]
```diff
@@ -11,6 +11,7 @@ Pod::Spec.new do |s|
     :tag => s.version
   }
   s.ios.deployment_target = '8.0'
+  s.osx.deployment_target = '10.9'
```
Comment (jpsim) [inline PeerKit.podspec]: MultipeerConnectivity is only available on 10.10. Thanks for the PR!

Comment (neonichu) [inline PeerKit.podspec]: Right, fixed.

**Comment by jpsim** [comment]
Woohoo! Thanks :tada:

**Comment by neonichu** [comment]
:beers:

**Pull request merged**